src/containers/Tenant/Diagnostics/Diagnostics.scss

```diff
@@ -1,3 +1,9 @@
+.yc-loader {
+    flex-grow: 1;
+    justify-content: center;
+    align-items: center;
```

Comment:
this works by accident
these properties only work because `.yc-loader` happen to have `display: flex` from the library styles
if it ever changes, this will break

Also I have a feeling that you used these in assumption that they would position `.yc-loader`, whereas in reality they control its content. Sorry if I'm wrong :)

so the simple way is to add `display: flex` to these styles (keeping in mind the other comment about selectors)
but maybe you want to revise the way this loader is positioned
either way is fine by me